src/main/java/org/sonarlint/intellij/config/SonarLintTextAttributes.java

```diff
@@ -40,11 +39,10 @@ public class SonarLintTextAttributes {
     /*
      * Defaults should be consistent with SonarLintSeverity
      */
-    CRITICAL = createTextAttributesKey("SONARLINT_CRITICAL", SonarLintSeverity.CRITICAL.defaultTextAttributes());
-    MAJOR = createTextAttributesKey("SONARLINT_MAJOR", SonarLintSeverity.MAJOR.defaultTextAttributes());
-    MINOR = createTextAttributesKey("SONARLINT_MINOR", SonarLintSeverity.MINOR.defaultTextAttributes());
-    INFO = createTextAttributesKey("SONARLINT_INFO", SonarLintSeverity.INFO.defaultTextAttributes());
-    BLOCKER = createTextAttributesKey("SONARLINT_BLOCKER", SonarLintSeverity.BLOCKER.defaultTextAttributes());
+    HIGH = createTextAttributesKey("SONARLINT_HIGH", SonarLintSeverity.HIGH.defaultTextAttributes());
+    MEDIUM = createTextAttributesKey("SONARLINT_MEDIUM", SonarLintSeverity.MEDIUM.defaultTextAttributes());
+    LOW = createTextAttributesKey("SONARLINT_LOW", SonarLintSeverity.LOW.defaultTextAttributes());
+    OLD_CODE = createTextAttributesKey("SONARLINT_OLD_CODE", SonarLintSeverity.OLD_CODE.defaultTextAttributes());
```

Comment:
I could try to retrieve previous keys by chaining them but for HIGH and LOW they are mapped to 2 severities (HIGH = CRITICAL & BLOCKER for example, so which key should we choose if they are different?)

I added this fallback mechanism for the MEDIUM impact -> `    MEDIUM = createTextAttributesKey("SONARLINT_MEDIUM", createTextAttributesKey("SONARLINT_MAJOR", SonarLintSeverity.MEDIUM.defaultTextAttributes()));
` since it's only one - one mapping

Reply:
But then if I do it only for once impact, maybe it's better to not do it at all? WDYT?

Reply:
Yes, I don't see a nice way to migrate this. Also, we don't even know if customization is used at all by users. I think we can just forget about old values